### PR TITLE
Hough transform float

### PR DESCRIPTION
**Pull request by 0x72D0**
PR for #216

### CODE REVIEW

File: src/math/hough_transform.cpp [hunk 1]
```diff
@@ -109,3 +110,18 @@ namespace Image_Function
         return true;
     }
 }
+
+namespace Image_Function
+{
+    bool HoughTransform( const std::vector< PointBase2D<double> > & input, double initialAngle, double angleTolerance, double angleStep,
+                         double lineTolerance, std::vector< PointBase2D<double> > & outOnLine, std::vector< PointBase2D<double> > & outOffLine )
+    {
+        runHoughTransform<double>(input, initialAngle, angleTolerance, angleStep, lineTolerance, outOnLine, outOffLine);
```
Comment (ihhub) [inline src/math/hough_transform.cpp]: You forgot to add return for this and below functions.

File: src/math/hough_transform.h [hunk 2]
```diff
@@ -3,9 +3,18 @@
...
 }
+
+namespace Image_Function
+{
+    bool HoughTransform( const std::vector< PointBase2D<double> > & input, double initialAngle, double angleTolerance, double angleStep,
+                         double lineTolerance, std::vector< PointBase2D<double> > & outOnLine, std::vector< PointBase2D<double> > & outOffLine );
+
+    bool HoughTransform( const std::vector< PointBase2D<float> > & input, float initialAngle, float angleTolerance, float angleStep,
+                         float lineTolerance, std::vector< PointBase2D<float> > & outOnLine, std::vector< PointBase2D<float> > & outOffLine );
+}
```
Comment (ihhub) [inline src/math/hough_transform.h]: Add an empty line at the end of this file.

**Comment by ihhub** [comment]
I'm trying to fix compilation warnings on Windows.

**Comment by 0x72D0** [comment]
there was a compilation error in linux, normally I fix it with this commit

**Comment by ihhub** [comment]
Cool. I'll check everything tomorrow (my tomorrow :) ).

**Comment by ihhub** [comment]
@0x72D0 there is some problem in unit tests based on *float*, they don't pass. Could you please take a look into it?

**Comment by 0x72D0** [comment]
ok I will check that

**Comment by ihhub** [comment]
Just to clarify: unit tests fail only on Windows.

**Comment by ihhub** [comment]
@0x72D0 thank you very much for working on and fixing this issue! Looking forward for SIMD implementation

**Pull request merged**